src/Microsoft.Health.Fhir.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -19,5 +19,10 @@ public class SqlServerDataStoreConfiguration
         /// If set, this applies schema 1 which resets all the data in the database. This is temporary until the schema migration tool is complete.
         /// </summary>
         public bool DeleteAllDataOnStartup { get; set; }
+
+        /// <summary>
+        /// Allows the experimental schema initializer to attempt to create the database if not present.
+        /// </summary>
+        public bool AllowDatabaseCreation { get; set; }
```

Comment:
To match current behavior I'm assuming we might want this turned on by default
```suggestion
        public bool AllowDatabaseCreation { get; set; } = true;
```

Reply:
Might not actually, the way i see it deployments are mostly likely to occur in Azure, the default behaviour would be that the database is already present. From a security standpoint it is not a good idea for an app to have access to a database via a privileged account that has permissions to create databases.

Reply:
Makes sense. What about developing locally, are there any additional steps there?

Reply:
We can definitely make it "visible" or explicit for local development, add this config property to both .Web (R4, Stu3) projects `launchSettings.json` file. Out of the box the functionality remains the same as before

```
"SqlServer": {
      "commandName": "Project",
      "environmentVariables": {
        ..........
        "SqlServer:Initialize": "true",
        "SqlServer:AllowDatabaseCreation": "true",
```